src/vs/editor/contrib/find/simpleFindWidget.ts

```diff
@@ -44,6 +45,22 @@ export abstract class SimpleFindWidget extends Widget {
 		this._findInput = this._register(new ContextScopedFindInput(null, this._contextViewService, {
 			label: NLS_FIND_INPUT_LABEL,
 			placeholder: NLS_FIND_INPUT_PLACEHOLDER,
+			validation: (value: string): InputBoxMessage | null => {
+				if (value.length === 0) {
+					return null;
+				}
+				if (!this._findInput.getRegex()) {
+					return null;
+				}
+				try {
+					/* tslint:disable:no-unused-expression */
+					new RegExp(value);
```

Comment:
Could be more simple use `tslint:disable-next-line:no-unused-expression` instead disable and enable next :slightly_smiling_face: